README.md

```diff
@@ -1,24 +1,167 @@
-application up and running.
+<div align="center">
+  <h3><b>Final-Capstone-back-end</b></h3>
+</div>
 
-Things you may want to cover:
+<!-- TABLE OF CONTENTS -->
 
-* Ruby version
+# 📗 Table of Contents
 
-* System dependencies
+- [📗 Table of Contents](#-table-of-contents)
+- [Hello world- ](#hello-world--)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run](#run)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Database creation
+# RailsREACT - <a name="about-project"></a>
 
-* Database initialization
+**HOTEL BOOKING** is the project built with Rails api to connect to the frontend react and redux. it's an application that allows users to add, delete and reserve hotels.
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with">
 
-* Services (job queues, cache servers, search engines, etc.)
+ruby on rails
+</a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+  </ul>
+  <summary>Client</summary>
+    <li><a href="https://github.com/felixDev22/Final-Capstone-front-end.git">FRONTEND</a><li>
+</details>
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- Key Features -->
+
+## Key Features <a name="key-features"></a>
+
+- Create an API endpint by rails.
+- Use api using webpack.
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+- Make sure you have Ruby installed in your system. You can install it [here](https://www.ruby-lang.org/en/documentation/installation/)
+
+- Get started with [Ruby on Rails](https://guides.rubyonrails.org/getting_started.html).
+
+- Make sure you have [PostgreSQL](https://www.postgresql.org/) installed and running.
+
+### Setup
+
+- Clone this repository by running `git clone https://github.com/elixDev22/Final-Capstone-back-end.git` in your command line.
+
+- Navigate to the repository by running `cd Final-Capstone-back-end`.
+
+### Install
+
+- Run `bundle install` to install all the gems.
+
+### Run
+
+- Change/include `host, username, password` according to your database server.
+- Run `rails db:create` and `rails db:migrate`
+- Run `rails s` to start the server.
+- In your browser, go to `http://localhost:3000`.
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+
+<!-- UML -->
```

Comment:
- [ ] You have a pretty functional and working `seeds.rb` on your project.  Kindly, tell your users to run "rails db:seed" so their database is filled with hotels so they can test and reserve. 💪